### DISCUSSION TITLE
Welcome to XC Discussions!

### DISCUSSION THREAD

**Discussion by lcpt** [xcfem/xc#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ebrahimraeyat** [comment]
Hello @lcpt, I am very happy for being in this community. I am a civil design engineer from Iran. We use ACI code for concrete and AISC for steel structure design.

Do your software support ACI318-2019 for design?

**Comment by lcpt** [comment]
Hi @ebrahimraeyat I think we have already talked some time ago, nice to meet you again.

About ACI318 we don't have implemented the checking routines for this code yet. As regards the AISC code there is already some work done you can take a look [here](https://github.com/xcfem/xc/tree/master/verif/tests/postprocess/limit_state_checking/aisc) and [here](https://github.com/xcfem/xc/tree/master/verif/tests/materials/astm_aisc).

At present, we are quite busy developing verification of steel connections using [CBFEM](https://www.cbfem.com/) and creating a module to import geometry from [FreeCAD](https://www.freecadweb.org/).

**Reply by ebrahimraeyat** [reply]
Thanks @lcpt yes we talked some time ago. I wanted to apply OpenSees solver in FreeCAD, related URL:

[OpenSees in FreeCAD](https://forum.freecadweb.org/viewtopic.php?f=18&t=20745&start=90)

And for CBFEM, I am very excited in Baseplate design:
[CBFEM FreeCAD](https://forum.freecadweb.org/viewtopic.php?f=18&t=42390)

can you show me an example of baseplate design in XC? Thank you so much

**Comment by lcpt** [comment]
Of course, I can. There it goes:

![baseplates_ULS02b_loads](https://user-images.githubusercontent.com/12873038/102335619-707cce00-3f90-11eb-9ed5-e5168680aa33.jpg)
![model](https://user-images.githubusercontent.com/12873038/102335667-812d4400-3f90-11eb-8df4-077bff43f5cc.png)
![von_mises_stress](https://user-images.githubusercontent.com/12873038/102335719-9013f680-3f90-11eb-95a0-f5c68a001afb.png)
![capacity_factor](https://user-images.githubusercontent.com/12873038/102335809-ade15b80-3f90-11eb-8f4e-bc89d814775e.png)
![weld](https://user-images.githubusercontent.com/12873038/102335838-b5086980-3f90-11eb-8eed-9bf6ed355cb1.png)

**Reply by ebrahimraeyat** [reply]
Wow, it seems it is very advanced analysis. Is this files is in test directory?
In several months ago I wanted to write a software in FreeCAD for designing Base Plate like this, you can download it for free:

[Base Plate Design with sap2000](https://sazeplus.com/product/auto-base/)

[Download](https://dl.sazeplus.com/do.php?filename=Auto-Base-v1-1-1-Free.rar)

**Comment by lcpt** [comment]
Good initiative, congratulations!

About the example of the images, we need to clean and reorganize the code before putting an example on the repository. Any help is welcome :)
